test/hummingbot/core/test_clock.py

```diff
@@ -0,0 +1,127 @@
+import unittest
+import asyncio
+import pandas as pd
+import time
+
+from hummingbot.core.clock import (
+    Clock,
+    ClockMode
+)
+from hummingbot.core.time_iterator import TimeIterator
+
+
+class ClockUnitTest(unittest.TestCase):
+
+    backtest_start_timestamp: float = pd.Timestamp("2021-01-01", tz="UTC").timestamp()
+    backtest_end_timestamp: float = pd.Timestamp("2021-01-01 01:00:00", tz="UTC").timestamp()
+    tick_size: int = 1
+
+    @classmethod
+    def setUpClass(cls):
+        cls.ev_loop: asyncio.BaseEventLoop = asyncio.get_event_loop()
+        return super().setUpClass()
```

Comment:
In the case of setUp methods, the call to super should be done as the first thing (because the superclass could be initializing things that are prerequisites for the initialization of the subclass).
It is correct to call super at the end in the tearDown method (since the cleanup is done in the reverse order from the setUp)